backend/tests/unit/services/test_agent_factory.py

```
@@ -496,4 +498,185 @@ def test_agent_import_failure_handling(self, mock_kubernetes_agent, mock_depende
+            llm_client=mock_dependencies['llm_client'],
+            mcp_client=mock_dependencies['mcp_client'],
+            mcp_registry=mock_dependencies['mcp_registry'],
+            agent_configs=agent_configs
+        )
+        
+        # Mock MCP registry
+        mock_dependencies['mcp_registry'].get_server_config.return_value = Mock()
+        mock_dependencies['mcp_registry'].get_server_configs.return_value = [Mock()]
+        
+        with caplog.at_level("INFO"):
+            agent = factory.create_agent("ConfigurableAgent:test-agent")
```

Ruff (0.12.2)

Local variable `agent` is assigned to but never used

Remove assignment to unused variable `agent`

(F841)